packages/signal_desktop.rb

```diff
@@ -3,12 +3,12 @@
 class Signal_desktop < Package
   description 'Private Messenger for Windows, Mac, and Linux'
   homepage 'https://signal.org/'
-  version '7.64.0'
+  version '7.66.0'
   license 'AGPL-3.0'
   compatibility 'x86_64'
-  min_glibc '2.29'
```

Comment:
Are you sure it works with the lower glibc versions?

Reply:
The highest `min_glibc` in this PR is 2.39, which is causing the unit tests to fail.

Break out that package, and I suspect the unit tests will pass.

Maybe building on the glibc 2.41 containers is broken.

Reply:
It makes sense to only use the newer glibc containers (which are much larger) when we need them since loading a smaller container makes everything go faster.

(Also, builds are much more reliable on the older containers.)

Reply:
You only need the min_glibc for packages that complain when installing in the lower glibc containers.


Realistically, nobody is running anything on systems with an older glibc in the real world.

Reply:
> Realistically, nobody is running anything on systems with an older glibc in the real world.

Well, apart from all the EOL devices...

Ideally we should try and keep as much compatibility with older devices as possible, especially since they don't really have an alternative.

Reply:
I still think the solution for THIS PR is to just separate out the packages into separate PRs, so that the unit tests don't fail.

The unit tests for the GUI apps are artificial in any case... and we're not actually testing the binaries to see if they work in the unit tests. (And when they DO complain, that's when we bump the `min_glibc` for that.

Essentially we have a `min_glibc` for build, and a `min_glibc` for running, and those are two entirely different things, and the builds from source DO test the former, and the unit tests SOMETIMES test the latter... but really not...

Reply:
Any package that depends on sommelier will fail because that doesn't run in containers.